### ISSUE TITLE
Ascii map plugin doesn't work by default, and can "leak" parts of map to the output window

### ISSUE THREAD

**Issue by fiendish** [fiendish/aardwolfclientpackage#24]
_From [aardrat...@gmail.com](https://code.google.com/u/112430013842115584456/) on December 10, 2010 02:03:34_

What steps will reproduce the problem? This may be two issues. Whatever.
A1. Download and run client. Map mini-window is visible, but doesn't show, even after playing a bit. Says it will appear after I start playing, but still doesn't show after I do a quest.

B1. After glancing quickly at the plugin code, I turn on "tags map on" and turn on "automap".
B2. The map works in the mini-window, but it appears to "leak" to the output window. What is the expected output? What do you see instead? (If applicable, include copy/paste from output) B2. See output window in attached screenshot. What version of the product are you using? On what operating system? Win 7 64

**Attachment:** [ascii_map_leak.png](http://code.google.com/p/aardwolfclientpackage/issues/detail?id=24)

_Original issue: http://code.google.com/p/aardwolfclientpackage/issues/detail?id=24_

**Comment by fiendish** [comment]
_From [aardrat...@gmail.com](https://code.google.com/u/112430013842115584456/) on December 09, 2010 23:05:42_

Note: I usually have automap off if I'm in another client

**Comment by fiendish** [comment]
_From [Fiendi...@gmail.com](https://code.google.com/u/103594094024149722366/) on December 19, 2010 09:05:46_

B1 is now fixed. B2 appears to be caused by one or more of Bast's plugins. Are you testing JUST this client?

**Status:** Fixed  
  **Labels:** -Type-Enhancement Type-Defect

**Comment by fiendish** [comment]
_From [aardrat...@gmail.com](https://code.google.com/u/112430013842115584456/) on December 19, 2010 09:35:48_

I did not have any of Bast's plugins installed. However, I may have had a few of my own authored plugins installed, and I have an idea of what may have caused it.

I did not have the B2 issue after I tried the new version perhaps a week ago, and that was definitely just the client.

So, probably not a bug on your end. Thanks.
